src/components/ReadingGoal/UpdateReadingGoalModal/index.tsx

```diff
@@ -119,18 +139,38 @@ const UpdateReadingGoalModal = ({ isDisabled, readingGoal }: UpdateReadingGoalBu
   return (
     <>
       <Button onClick={onUpdateGoalClicked} isDisabled={isDisabled}>
-        {t('edit-goal')}
+        {t('edit-goal.action')}
       </Button>
       <Modal isOpen={isModalVisible} onClickOutside={closeModal}>
         <Modal.Body>
           <Modal.Header>
-            <Modal.Title>{t('delete-confirmation.title')}</Modal.Title>
-            <Modal.Subtitle>{t('delete-confirmation.subtitle')}</Modal.Subtitle>
+            <Modal.Title>{t('edit-goal.title')}</Modal.Title>
+            <Modal.Subtitle>{t('edit-goal.subtitle')}</Modal.Subtitle>
+
+            {/* <div className={styles.radioGroup}> */}
```

Comment:
Nice we should remove this `{/* <div className={styles.radioGroup}> */}` and the closing tag too.